### PR TITLE
Spaceship Turning Speed

### PR DESCRIPTION
**Pull request by 3MKJOE58**
## Describe your changes

Changing spaceship turning speed 
Adding slider to setting menu to control mouse sensitivity speed

## Issue number and link

#208

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have written any applicable tests
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] This complies with the [product coding guidelines](https://github.com/BIT-Studio-3/Space-Rescue/wiki/Code-Review-Format)

### CODE REVIEW

**Review by PBWiseman — CHANGES_REQUESTED** [review]
Definitely an improvement! I noticed in the code the value of mouseSensitivity is .3f and in the editor it is .5f. Which one was intended? .5f still feels a little fast personally but I'd like to hear others opinion on it.

**Review by erokasyuart — CHANGES_REQUESTED** [review]
I'm somebody who plays with mouse sensitivity turned all the way up. Personally, this feels very very slow and hard to turn, but others might think it's too fast. I think the resolution to this is to implement something that is a spectrum, so it fits all users' needs. Some people have said the flying is very motion sickness-inducing, but for me, I couldn't care less! I think it would be most useful to add a slider into the settings menu that changes the value! I think sliders are fairly simple to understand, I don't think it will take you long at all!

**Comment by PBWiseman** [comment]
> I'm somebody who plays with mouse sensitivity turned all the way up. Personally, this feels very very slow and hard to turn, but others might think it's too fast. I think the resolution to this is to implement something that is a spectrum, so it fits all users' needs. Some people have said the flying is very motion sickness-inducing, but for me, I couldn't care less! I think it would be most useful to add a slider into the settings menu that changes the value! I think sliders are fairly simple to understand, I don't think it will take you long at all!

Oh a slider is a really good idea! Seconded!

**Review by ChaseBENNC9 — CHANGES_REQUESTED** [review]
I think it is definitely an improvement. But i agree with other reviewers in that it is slightly slow. And I agree that a slider would provide a wide range of options for different players

File: Assets/Scripts/Ship Movement/ShipMovement.cs [hunk 1]
```diff
@@ -41,7 +42,6 @@ public class ShipMovement : MonoBehaviour
     void Start()
     {
         Instance = this;
-        Cursor.lockState = CursorLockMode.Locked; //keep mouse in the game
```
Comment (erokasyuart) [inline Assets/Scripts/Ship Movement/ShipMovement.cs:44]: Add this back in! At the very start of the game, when main scene is first loaded, the mouse is not locked.

**Review by erokasyuart — CHANGES_REQUESTED** [review]
Good improvement to the movement. Highly adjustable for players preferences.

Just some slight tweaks needed:
image
Probably best to remove "volume" from Mouse Sensitivity Text

**Review by PBWiseman — CHANGES_REQUESTED** [review]
Great work. The slider works properly and the sensitivity changes. Minor tweaks needed:
Can there please be a minimum sliver value set? This can be done on the slider game object and make sure that the player doesn't reduce the value to 0 and not be able to fly.
The text for mouse sensitivity slider should probably remove the Volume part as it isn't really a volume.
The mouse sensitivity slider is a little bit out of alignment with the other settings. If you set the Y position of all the objects inside of the Mouse Sensitivity parent (text, slider, and number) and change the y position of the parent to -140 to continue the 70 gap then that would look more even.

**Review by ChaseBENNC9 — CHANGES_REQUESTED** [review]
Slider works great definitely allows players to customize their experience. Once thing I noticed was you can set the slider to 0 which isn't something you'd expect in a sensitivity slider. An easy way to change this is in the sensitivity slider in the settings scene and changes min from 0 to to 0.1,

**Review by ChaseBENNC9 — APPROVED** [review]

**Review by PBWiseman — APPROVED** [review]

**Review by erokasyuart — APPROVED** [review]
Swagalicious!

**Pull request merged**